src/invidious/config.cr

```diff
@@ -100,7 +100,7 @@ class Config
   property domain : String?
   # Subscribe to channels using PubSubHubbub (requires domain, hmac_key)
   property use_pubsub_feeds : Bool | Int32 = false
-  property popular_enabled : Bool = true
+  property pages_enabled : Hash(String, Bool) = {"trending" => true, "popular" => true, "search" => true}
```

Comment:
I think the primary blocker for this PR right now is that this is a breaking change. I'll suggest to keep `popular_enabled` for now but maybe with a deprecation warning. 

Some clear precedence of the two config options should also be set and documented to handle the case of someone configuring both of them at the same time.